18F/identity-idp
# Bump rexml to resolve security advisory

## 🛠 Summary of changes

Updates `rexml` to resolve a security advisory.

```
Name: rexml
Version: 3.3.1
CVE: CVE-2024-39908
Criticality: Unknown
URL: https://github.com/ruby/rexml/security/advisories/GHSA-4xqq-m2hx-25v8
Title: DoS in REXML
Solution: upgrade to '>= 3.3.2'
```

## 📜 Testing Plan

1. Run `make audit`
2. Observe no errors in output
